lib/tls.js

```diff
@@ -163,6 +163,33 @@ function check(hostParts, pattern, wildcards) {
   return true;
 }
 
+exports._canonicalIp = function(address) {
```

Comment:
I'd generally prefer not introducing new `_`-prefixed methods. Non-public methods should be moved into `lib/internal/` and used from there.